Tests/Mocks/PaymentViewControllerMockDelegate.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  PaymentViewControllerMockDelegate.swift
+//  FramesTests
+//
+//  Created by Ehab Alsharkawy.
+//  Copyright © 2022 Checkout. All rights reserved.
+//
+
+import UIKit
+import Checkout
+@testable import Frames
+
+class PaymentViewControllerMockDelegate: PaymentViewControllerDelegate {
+
+  var addBillingButtonIsPressedWithSender: [UINavigationController?] = []
+  var editBillingButtonIsPressedWithSender: [UINavigationController?] = []
+  var expiryDateIsUpdatedWithValue: [ExpiryDate?] = []
+  var securityCodeIsUpdatedWithValue: [String?] = []
+  var cardNumberIsUpdatedWithValue: [String?] = []
```

Comment:
Question : Why is this a array containing card numbers. I can see `delegate?.cardNumberIsUpdated(value: cardNumber)` is called which passes the cardNumber and its appending in array.